tests/e2e/testfiles/storage_csm_powerscale_observability.yaml

```diff
@@ -1,488 +1,488 @@
-apiVersion: storage.dell.com/v1

```

Comment:
This is another 500 line difference; what did this change encompass? 

This PR is so large that it is crashing my browser trying to go through the diffs. If all of this is a result of YAML linting, then we should have rolled out those changes incrementally, shouldn't we?